README.md

```diff
@@ -4,6 +4,16 @@ This is the website for DeXter On Radix, a decentralized exchange on the [Radix
 
 ## Getting Started
 
+Set Up:
+
+- Fork the Dexter repository to your github account.
+- Clone from your forked repo.
+- Make a new branch with feature name(can see example in some [closed PRs](https://github.com/DeXter-on-Radix/website/pulls?q=is%3Apr+is%3Aclosed)).
+- Make some changes.
+- Commit and push local branch changes to your own repo.
+- Make a pull request into the Dexter Repo(original).
+
+Now you ready to run locally the project(this will show in your default browser as a LOCALHOST)
 To run the website locally:
```

Comment:
```suggestion
```

Reply:
After:

```bash
npm i
npm run dev
```

you can add a comment that there will be a link to the locally served instance in the console output. Don't specify exact link (like `http://localhost:3000`) because if this port is busy, it'll start on another.

Reply:
Will remind unchanged in README.md